site.ts

```diff
@@ -409,6 +409,17 @@ export function inject({ config, posthog }) {
         const nonAPISurveys = surveys.filter(survey => survey.type !== 'api')
         nonAPISurveys.forEach((survey) => {
             if (document.querySelectorAll("div[class^='PostHogSurvey']").length === 0) {
+                const surveyWaitPeriodInDays = survey.conditions?.seenSurveyWaitPeriodInDays
```

Comment:
this could really use some tests 😅

Reply:
imports are being kind of annoying for tests, I'll add this in a separate PR to avoid breaking everything 😅